docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -369,6 +369,8 @@ Save this file in `src/main/docker/Dockerfile.multistage` as it is not included
 
 [WARNING]
 ====
+Replace the `-java11` in the base image with `-java8` if you want to use Java 8.
```

Comment:
As mentioned by Loïc, let's do the opposite for now.

We will migrate to Java 11 by default soon but right now we still use java8 as the default for native images.